webapp/src/components/modals/channel_settings/edit_channel_settings.tsx

```diff
@@ -62,20 +62,6 @@ export type State = {
     conflictingError: string | null;
 };
 
-const removeDuplicateEvents = (array: ReactSelectOption[]): ReactSelectOption[] => {
```

Comment:
This was a solution for what we thought was the problem, when really the problem was that we were introducing duplicate default fields in the dataset, instead of just custom fields.